app/page.tsx

```diff
@@ -1,7 +1,15 @@
+"use client";
+
+import SearchBar from "@/components/SearchBar";
```

Comment:
notice this import if you haven't already!

this is a typescript path alias, where `@` is the root of your directory ([it's defined in your tsconfig.json](https://github.com/Rmaki27/collections/blob/main/tsconfig.json#L23))

super helpful when you want to access components or modules closer to your root when you're in a deeply nested file.

Reply:
Super interesting thanks for the info!